IlmBase/IlmThread/IlmThreadSemaphoreWin32.cpp

```diff
@@ -37,8 +37,11 @@
 //	class Semaphore -- implementation for Windows
 //
 //-----------------------------------------------------------------------------
+#if (defined(_WIN32) || defined(_WIN64))
```

Comment:
HAVE_PTHREAD guard?

Reply:
As I said earlier, if pthread is not actually available, this should just include windows.h.